pkgs/applications/kde/kaddressbook.nix

```
@@ -21,4 +21,7 @@ mkDerivation {
     kdbusaddons ki18n kontactinterface kparts kpimtextedit
     kxmlgui libkdepim libkleo mailcommon pimcommon prison qgpgme qtbase
   ];
+  postInstall = ''
+    mkdir -p "$out/include/KF5"
+  '';
```

Why do we need this?

---

I wanted to comment better on this, but didn't want to pollute more the log. Not sure if some compilation or cmake flag of kaddressbook's 22.04 release, or the gcc11 bump, but the build system (of `kdepim-addons` iirc) started erroring if some `-I`d directory don't exist. kaddressbook includes `/include/KPim` and `/include/KF5` but only the former exists, and this errors the whole build in NixOS (not in other OSs, as KF5 exists for plenty of packages there). The easiest solution was to simply create the empty directory. I thought it was easier than finding the CMakeLists with include to `substituteInplace`

---

```suggestion
  postInstall = ''
    # added as an include directory by other packages which fail to compile if the it is missing
    mkdir -p "$out/include/KF5"
  '';
```
or something like this